### PR TITLE
Programs text align and Exhibit meta removal

### PR DESCRIPTION
**Pull request by madsad87**
Centered text in the Program blocks on the /programs page and removed the visible date from the /exhibit archive page

## JIRA Ticket
[THNK-23](https://wpengine.atlassian.net/projects/THNK/issues/THNK-23)

## Description

This change center aligns the text in the Program boxes on the /programs page as well as removes post meta specifically for the date of publishing on the individual exhibits from the /exhibit archive page.

## Impacted Areas in Application

List general components of the site that this PR will affect:

thinkery.wpengine.com/programs

`.program-buttons`

thinkery.wpengine.com/exhibit

`entry-meta`

## Deploy Notes

Notes regarding deployment the contained body of work.
If you need to change something in WordPress after deploying, make the necessary steps crystal clear.

## Steps to Test or Reproduce

1. pull down branch
2. Load up the thinkery.wpengine.com/exhibit page and you should notice that the date isn't displaying on the Exhibit boxes.
3. Load up the thinkery.wpengine.com/programs page and the text will be center aligned at all window sizes

## Random GIF

Include one random GIF to "pay" your reviewers in humor.

Thanks for reviewing my code!

### CODE REVIEW

File: wp-content/themes/thinkery/style.css [hunk 1]
```diff
@@ -221,6 +221,10 @@ blockquote {
 	margin: 30px;
 }
 
+.program-buttons {
+	test-align: center;
```
Comment (sarahricker) [inline wp-content/themes/thinkery/style.css]:
```suggestion
	text-align: center;
```

Is this necessary? Can't we center through the block settings / general WP justification classes?

File: wp-content/themes/thinkery/style.css [hunk 2]
```diff
@@ -221,6 +221,10 @@ blockquote {
...
+}
+
```
Comment (sarahricker) [inline wp-content/themes/thinkery/style.css]: Actually, if we're going this route with the new buttons, lets remove this class use the existing `has-text-align-center`core class instead. Its not an option in the settings for the button being used (and we can't filter 3rd party blocks TMK), but is more global / adds less weight & confusion to the style.css.

File: wp-content/themes/thinkery/functions.php [hunk 3]
```diff
@@ -641,3 +641,12 @@ function add_file_types_to_uploads($file_types){
...
 add_filter('upload_mimes', 'add_file_types_to_uploads');
+
+// Remove Post Info, Post Meta from /exhibit Archive Pages
+add_filter( 'genesis_post_info', 'remove_archive_cpt_post_info' );
+function remove_archive_cpt_post_info($post_info) {
+if ( is_post_type_archive('exhibit') ) :
+	$post_info = '[post_comments] [post_edit]';
+	return $post_info;
+	endif;
+}
```
Comment (sarahricker) [inline wp-content/themes/thinkery/functions.php]: This seems good, can we maybe move it up to [#L277](https://github.com/sarahricker/thinkery/blob/269051b4d3c8979dfbe12f4ad6f2c01c7b616f5d/wp-content/themes/thinkery/functions.php#L277) so it lives near all the other archive adjustments?

**Review by sarahricker — CHANGES_REQUESTED** [review]
@madsad87 Looks good, just some small changes notated in the code. 

Re programs page: Personally, I'd rather see the whole cover block change overlay color on hover + link to the page like the old slideshows did,  as this would prevent text wrapping within the buttons. 

Speaking of buttons, they also don't seem to match our main button styles, wondering if there is room to match them closer ( originally `padding: 20px 64px;` ) - though that would likely increase the wrapping at more breakpoints 😢 

Thoughts?

File: wp-content/themes/thinkery/functions.php (hunk 3 above)
Comment (madsad87) [reply wp-content/themes/thinkery/functions.php]: Yep, good call on relocating the code. Will do

File: wp-content/themes/thinkery/style.css [hunk 4]
```diff
@@ -221,6 +221,7 @@ blockquote {
 	margin: 30px;
 }
 
+
```
Comment (sarahricker) [inline wp-content/themes/thinkery/style.css]:
```suggestion

```

**Review by sarahricker — APPROVED** [review]
LGTM! Cleaned up the spacing, willl merge and deploy

**Pull request merged**